frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -33,6 +33,42 @@ const DataFormat: FC<Props> = ({
     handleChangeRaw(value);
   };
 
+  const renderH5adRadio = (): React.ReactElement => {
+    return (
+      <Radio
+        disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.H5AD)}
+        label=".h5ad (AnnData v0.7)"
+        value={DATASET_ASSET_FORMAT.H5AD}
+      />
+    );
+  };
+
+  const renderRdsRadio = (): React.ReactElement => {
+    return (
+      <Radio
+        disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.RDS)}
+        label=".rds (Seurat v3)"
+        value={DATASET_ASSET_FORMAT.RDS}
+      />
+    );
+  };
+
+  const renderDisabledRdsRadio = (): React.ReactElement => {
+    return (
+      <Tooltip
+        disabled={false}
+        interactionKind={PopoverInteractionKind.HOVER}
+        content="A .rds (Seurat v3) download is unavailable due to limitations in the R dgCMatrix sparse matrix class."
+        intent={Intent.DANGER}
+        position={Position.TOP}
+      >
+        {/* Logically renders only when rds format not available, but a Tooltip wrapper also happens to prevent 
+        proper functioning of radio buttons with a larger radio group outside of the Tooltip wrapper */}
```

Comment:
☝️ ⬆️ 🔑 

This is the main issue. Rearranged rendering logic to make sure the rds radio button isn't wrapped when it is not disabled.

Reply:
👎 BP3